### PR TITLE
Features branch

### PR DESCRIPTION
**Pull request by sunga12**
Pages Built:
- The home page.
- The about page.

For both the Mobile Version & Desktop Version.

- Interactions:
1. The home page has a link in the menu to the about page.
2. The logo in the header links to the home page.
3. When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
4. Mobile menu is implemented so it is consistent with the design (colors, typography, spacing, etc.).
5. The mobile menu should has a close (X) button that closes the menu.

- Dynamic page

1. The section "Featured speakers" was created dynamically in JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          <div class="feature-item"><p>Fairness</p></div>
+          <div class="feature-desc"><p>We treat our pupils, employees and all stakeholders in a fair and respectful manner.</p></div>
+        </div>
+        
+        <div class="school-feature">
+          <div class="feature-icon"><img src="./images/progress.png" alt=""></div>
+          <div class="feature-item"><p>Progress</p></div>
+          <div class="feature-desc"><p>We strive for continuous improvement, advancing our capacity and facilities to serve future generations.</p></div>
+        </div>
+
```
Comment (mwanawabangona) [inline index.html:80]: - [ ] `Required`
- [ ] Amazing work so far 👏 
- [ ] However there is a few thing 🤏 you to adjust.
- [ ] Kindly use the background used the original design.
- [ ] please see screenshots below 🥂 

`your design`
image

`origin design`
image

File: README.md [hunk 2]
```diff
@@ -207,82 +162,45 @@ Example:
...
+- GitHub: [@sunga12](https://github.com/sunga12)
+- Twitter: [@OfficialAseT](https://twitter.com/OfficialAseT)
+- LinkedIn: [Sungabanja Thawethe](https://www.linkedin.com/in/sungabanja-thawethe-b3419b142/)
 
 👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Cindy Shin on Behance: (https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```
Comment (mwanawabangona) [inline README.md]: - [ ] `Required`
- [ ] Kindly add this cindyshin at the bottom next the licence 🥂 😉

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @sunga12 ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sunga12** [comment]
Changes made:

- Changed background of main program section\
- moved Cindy shin license to license section

File: README.md [hunk 3]
```diff
@@ -1,40 +1,16 @@
...
-
-After you're finished please remove all the comments and instructions!
--->
+
 
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+
+  <img src="./images/school-logo-small.png" alt="logo" width="140"  height="auto" />
```
Comment (Lucas-Masaba) [inline README.md]: [OPTIONAL] 
- The logo on your README.md is not showing up as seen in the image below.
image

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @sunga12  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- All changes were made as requested by the previous reviewer 🏄 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**